python/cugraph/community/louvain_wrapper.pyx

```diff
@@ -79,28 +79,27 @@ def louvain(input_graph, max_level, resolution):
     # FIXME: The excessive casting for the enum arg is needed to make cython
     #        understand how to pass the enum value (this is the same pattern
     #        used by cudf). This will not be needed with Cython 3.0
-    populate_graph_container(graph_container,
-                             <legacyGraphTypeEnum>(<int>(legacyGraphTypeEnum.CSR)),
-                             handle_[0],
-                             <void*>c_offsets, <void*>c_indices, <void*>c_weights,
-                             <numberTypeEnum>(<int>(numberTypeEnum.intType)),
-                             <numberTypeEnum>(<int>(numberTypeEnum.intType)),
-                             <numberTypeEnum>(<int>(weightTypeMap[weights.dtype])),
-                             num_verts, num_edges,
-                             <int*>c_local_verts, <int*>c_local_edges, <int*>c_local_offsets,
-                             False, True)  # store_transposed, multi_gpu
-
-    graph_container.get_vertex_identifiers(<void*>c_identifier)
+    populate_graph_container_legacy(graph_container,
+                                    <legacyGraphTypeEnum>(<int>(legacyGraphTypeEnum.CSR)),
+                                    handle_[0],
+                                    <void*>c_offsets, <void*>c_indices, <void*>c_weights,
+                                    <numberTypeEnum>(<int>(numberTypeEnum.intType)),
+                                    <numberTypeEnum>(<int>(numberTypeEnum.intType)),
+                                    <numberTypeEnum>(<int>(weightTypeMap[weights.dtype])),
+                                    num_verts, num_edges,
+                                    <int*>c_local_verts, <int*>c_local_edges, <int*>c_local_offsets)
```

Comment:
What happens if vertex_t is int64_t? Is this path irrelevant or it's not supported?

Reply:
To be addressed in next PR (see comment above).

Reply:
Looked closer at adding support for int64_t and talked with @ChuckHastings and realized we wouldn't support int64_t vertex types for legacy graph classes.